sycl/source/detail/stream_impl.hpp

```diff
@@ -35,8 +35,8 @@ class __SYCL_EXPORT stream_impl {
   // buffer and offset in the flush buffer
   GlobalOffsetAccessorT accessGlobalOffset(handler &CGH);
 
-  // Copy stream buffer to the host and print the contents
-  void flush();
+  // Enqueue task to copy stream buffer to the host and print the contents
+  void enqueueFlush();
```

Comment:
I think we should not break ABI here.

```suggestion
  // will merely call enqueueFlush and wait for the event.
  void flush();
  // Enqueue task to copy stream buffer to the host and print the contents
  event enqueueFlush();
```

Reply:
Agree, changed to old name.